platform/broadcom/one-image.mk

```diff
@@ -34,7 +34,8 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
                                $(QUANTA_IX1B_32X_PLATFORM_MODULE) \
                                $(MITAC_LY1200_32X_PLATFORM_MODULE) \
                                $(ALPHANETWORKS_SNH60A0_320FV2_PLATFORM_MODULE) \
-                               $(ALPHANETWORKS_SNH60B0_640F_PLATFORM_MODULE)
+                               $(ALPHANETWORKS_SNH60B0_640F_PLATFORM_MODULE) \
+                               $(CEL_SEASTONE2_PLATFORM_MODULE)
```

Comment:
Please move '$(CEL_SEASTONE2_PLATFORM_MODULE)' to line number 29 to join with other celestica devices